presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TopNMatcher.java

```diff
@@ -17,32 +17,28 @@
 import com.facebook.presto.cost.StatsProvider;
 import com.facebook.presto.metadata.Metadata;
 import com.facebook.presto.spi.block.SortOrder;
+import com.facebook.presto.sql.planner.OrderingScheme;
 import com.facebook.presto.sql.planner.Symbol;
 import com.facebook.presto.sql.planner.plan.PlanNode;
 import com.facebook.presto.sql.planner.plan.TopNNode;
-import com.google.common.base.Functions;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Maps;
-
-import java.util.List;
-import java.util.Map;
+import com.google.common.collect.ImmutableMap;
 
 import static com.facebook.presto.sql.planner.assertions.MatchResult.NO_MATCH;
 import static com.facebook.presto.sql.planner.assertions.MatchResult.match;
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static com.google.common.base.Preconditions.checkState;
-import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.Objects.requireNonNull;
 
 public class TopNMatcher
         implements Matcher
 {
     private final long count;
-    private final List<PlanTestSymbol> orderBySymbols;
+    private final ImmutableMap<String, SortOrder> orderingScheme;
```

Comment:
`Map`

Reply:
ditto, we care about key insertion order.